internal/builtinfunctions/functions_test.go

```diff
@@ -819,3 +821,100 @@ func Test_floatToFormattedString_success(t *testing.T) {
 		}
 	}
 }
+
+func Test_bindConstants(t *testing.T) {
```

Comment:
This is a good "happy test".  Are there any other cases that we should test?  Different types (e.g., scalar or list) for the repeated input, or a zero-length "items" list?

Reply:
yeah I can't think of any reason to allow zero element lists; @jaredoconnell what do you think the expected behavior should be here?

Reply:
added it